migrate_to_auto_rag.sh

```
@@ -0,0 +1,84 @@
+#!/bin/bash
+# migrate_to_auto_rag.sh - One-click migration script
+
+set -e
+
+echo "🚀 Starting Auto-Discovery RAG Migration..."
+
+# Create backup of current system
+echo "📦 Creating backup..."
+mkdir -p backups/$(date +%Y%m%d_%H%M%S)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)